src/components/CardImage/CardImage.js

```diff
@@ -0,0 +1,17 @@
+import "./CardImage.css";
+import defaultImage from "../../assets/images/card-default.png";
+
+const CardImage = ({ items, isZoomedIn }) => {
```

Comment:
P1:
prop명이 items 가 아닌 item이 더 이해하기 쉬울 것 같아요. 또한 items에서 imageSource 만 필요하니 해당 값만 받으면 더 좋겠네요.
```suggestion
const CardImage = ({ imageSource, isZoomedIn }) => {
```